core/src/consensus/tendermint/worker.rs

```diff
@@ -2123,6 +2154,7 @@ impl Worker {
                 if let Err(double_vote) = self.votes.collect(vote) {
                     cerror!(ENGINE, "Double vote found on_commit_message: {:?}", double_vote);
                     self.report_double_vote(&double_vote);
```

Comment:
I restored `report_double_vote` since currently, the evidences are not handled.
Let's remove it after the client uses evidences.

Reply:
okay